util/raclgen/lib.py

```diff
@@ -50,3 +57,51 @@ def compute_policy_value(permission: str) -> int:
             policy['rd_default'] = compute_policy_value('allowed_rd')
             policy['wr_default'] = compute_policy_value('allowed_wr')
     return racl_config
+
+
+def parse_racl_mapping(racl_config: Dict[str, object], mapping_path: str,
+                       if_name: Optional[str], ip_block: IpBlock):
+    mapping = _read_hjson(mapping_path)
+    parsed_register_mapping = {}
+
+    # Mapping must be a dict with a single entry:
+    # RACL_GROUP => register mapping
+    if len(mapping) != 1:
+        raise SystemExit('Mapping file must be a single-element dict mapping the RACL group to the '
+                         'register mapping')
+    racl_group, register_mapping = list(mapping.items())[0]
+
+    if not isinstance(racl_group, str):
+        raise SystemExit('RACL group must be a string')
+    if not isinstance(register_mapping, dict):
+        raise SystemExit('Register mapping must be a a dict')
+
+    # Special handling of the all star assignment:
+    # "*": POLICY
+    # Assigns all registers to a given policy
+    if list(register_mapping.keys()) == ["*"]:
+        policy_name = list(register_mapping.values())[0]
```

Comment:
I agree that this is correct, but `policy_name = register_mapping["*"]` is probably more obvious?